File: Java/refactoring_exercise_final/src/test/java/za/co/entelect/refactoring_final/controller/ImageControllerTest.java
package za.co.entelect.refactoring_final.controller;

import org.junit.Test;
import za.co.entelect.refactoring_final.service.ImageService;

import static junit.framework.Assert.assertNotNull;

public class ImageControllerTest {

    private ImageController imageController = new ImageController();

    @Test
    public void testFetchImage(){
        assertNotNull(imageController.fetchImage(ImageService.DEFAULT_IMAGE));
    }

    @Test
    public void testUploadImage(){
        assertNotNull(imageController.uploadImage("1", new byte[1]));
    }
}

File: C#/Refactoring/refactoring_exercise_final/test/za/co/entelect/refactoring_final/controller/ImageControllerTest.cs
using NUnit.Framework;
using refactoring_exercise_final.za.co.entelect.refactoring_final.controller;
using refactoring_exercise_final.za.co.entelect.refactoring_final.service;

namespace refactoring_exercise_final.test.za.co.entelect.refactoring_final.controller
{
    public class ImageControllerTest
    {
        private readonly ImageController _imageController = new ImageController();

        [Test]
        public void TestFetchImage(){
            Assert.NotNull(_imageController.FetchImage(ImageService.DefaultImage));
        }

        [Test]
        public void TestUploadImage(){
            Assert.NotNull((_imageController.UploadImage("1", new byte[1])));
        }
    }
}